lib/galaxy/datatypes/isa.py

```diff
@@ -440,32 +403,6 @@ def groom_dataset_content(self, file_name):
             logger.debug("Files in %s after grooming...", output_path)
```

Comment:
These more complex debug logging sections can be wrapped inside `if logger.isEnabledFor(logging.DEBUG):` to avoid having the program needlessly go through the loop and the `os.listdir` when the resulting logging messages would be filtered.